processes/loc.py

```diff
@@ -0,0 +1,208 @@
+import json
+import os, requests
+from requests.exceptions import HTTPError, ConnectionError
+
+from .core import CoreProcess
+from mappings.core import MappingError
+from mappings.loc import LOCMapping
+from managers import WebpubManifest
+from logger import createLog
+
+logger = createLog(__name__)
+
+LOC_ROOT_OPEN_ACCESS = 'https://www.loc.gov/collections/open-access-books/?fo=json&fa=access-restricted%3Afalse&c=25&at=results'
+LOC_ROOT_DIGIT = 'https://www.loc.gov/collections/selected-digitized-books/?fo=json&fa=access-restricted%3Afalse&c=25&at=results' 
+
+class LOCProcess(CoreProcess):
+
+    def __init__(self, *args):
+        super(LOCProcess, self).__init__(*args[:4])
+
+        self.ingestOffset = int(args[5] or 0)
+        self.ingestLimit = (int(args[4]) + self.ingestOffset) if args[4] else 5000
+        self.fullImport = self.process == 'complete' 
+
+        # Connect to database
+        self.generateEngine()
+        self.createSession()
+
+        # Connect to epub processing queue
+        # self.fileQueue = os.environ['FILE_QUEUE']
+        # self.fileRoute = os.environ['FILE_ROUTING_KEY']
+        # self.createRabbitConnection()
+        # self.createOrConnectQueue(self.fileQueue, self.fileRoute)
+
+        # S3 Configuration
+        self.s3Bucket = os.environ['FILE_BUCKET']
+        self.createS3Client()
+
+    def runProcess(self):
+        count = 0
+        count2 = 0
```

Comment:
Might be more readable to name these according to what they're counting, ie, `openAccessRequestCount` and `digitizeRequestCount`?

Reply:
Sounds good